src/snowcli/cli/connection/commands.py

```diff
@@ -188,11 +188,11 @@ def add(
 
 
 @app.command()
-@global_options
+@global_options_with_connection
 @with_output
-def test(connection: str = ConnectionOption, **options) -> CommandResult:
+def test(**options) -> CommandResult:
     """
     Tests the connection to Snowflake.
     """
-    connect_to_snowflake(connection_name=connection)
+    connect_to_snowflake(**options)
```

Comment:
1. IMO we should rewrite `connect_to_snowflake` arguments manually.
I don't like introducing some silent contract between that function and the command options.

2. If you agree, then IMO we should also revert old `connection_name` function's argument name.

3. Can't we use `get_connection` from global context with `force`?